development/compiling/introduction_to_the_buildsystem.rst

```diff
@@ -124,63 +124,56 @@ you can build them yourself).
-manager.
+Target controls if the editor is contained and used debug flags.
+All builds are optimized. Each mode means:
 
-::
+-  **editor**: Build with editor, optimized, with debugging code
+-  **template_debug**: Build with C++ debugging symbols
+-  **template_release**: Build without symbols
 
-    scons platform=<platform> tools=yes/no
+The editor is enabled by default in all PC targets (Linux, Windows, macOS),
+disabled for everything else. Disabling the editor produces a binary that can
+run projects but that does not include the editor or the project manager.
 
-.. _doc_introduction_to_the_buildsystem_target:
+::
 
-Target
-------
+    scons platform=<platform> target=editor/template_debug/template_release
 
-Target controls optimization and debug flags. Each mode means:
+Dev build
+---------
 
--  **debug**: Build with C++ debugging symbols, runtime checks (performs
-   checks and reports error) and none to little optimization.
--  **release_debug**: Build without C++ debugging symbols and
-   optimization, but keep the runtime checks (performs checks and
-   reports errors). Official editor binaries use this configuration.
--  **release**: Build without symbols, with optimization and with little
-   to no runtime checks. This target can't be used together with
-   ``tools=yes``, as the editor requires some debug functionality and run-time
-   checks to run.
+When doing enigine development the ``dev_build`` option can be used together
+with ``target`` to enable dev-specific code.
 
 ::
 
-    scons platform=<platform> target=debug/release_debug/release
+    scons platform=<platform> dev_build=yes
 
-This flag appends the ``.debug`` suffix (for debug), or ``.tools`` (for debug
-with tools enabled). When optimization is enabled (release), it appends
-the ``.opt`` suffix.
+This flag appends the ``.dev`` suffix (for development) to the generated
+binary name.
 
-Bits
-----
+Architecture
+------------
 
-Bits is meant to control the CPU or OS version intended to run the
+The ``arch`` option is meant to control the CPU or OS version intended to run the
 binaries. It is focused mostly on desktop platforms and ignored everywhere
 else.
 
--  **32**: Build binaries for 32-bit platforms.
--  **64**: Build binaries for 64-bit platforms.
--  **default**: Build for the architecture that matches the host platform.
+Supported values for the ``arch`` option are **auto**, **x86_32**, **x86_64**,
+**arm32**, **arm64**, **rv64**, **ppc32**, **ppc64** and **wasm32**.
 
 ::
 
-    scons platform=<platform> bits=default/32/64
+    scons platform=<platform> arch=auto/'auto/x86_32/x86_64/arm32/arm64/rv64/ppc32/ppc64/wasm32
 
-This flag appends ``.32`` or ``.64`` suffixes to resulting binaries when
-relevant. If ``bits=default`` is used, the suffix will match the detected
-architecture.
+This flag appends the value of ``arch`` to resulting binaries when
+relevant.  The default value ``arch=auto`` builds for the architecture
```

Comment:
The default value ``arch=auto`` detects the architecture that matches the host platform. 
Sounds a bit better here.